### ISSUE TITLE
Colourscheme: Allow init.vim value to be used.

### ISSUE THREAD

**Issue by CrossR** [onivim/oni#2211]
Currently, as far as I can tell at least, there is no built in way of defering the UI colour scheme to use values defined in the user's `init.vim`. We rely on the user instead setting `ui.colorscheme` to something, either an Oni or a Vim theme.

It would be nice if we could instead allow the user to use their `init.vim` theme, such that they don't have to define things twice. The "workaround" I've got so far is just setting `"ui.colorscheme": "n/a",` such that Oni fails to load any colorscheme. Setting it to any blank value just uses the `DefaultTheme` setting which sets the `default` color scheme.

It'd instead be better if we had an explicit setting or option for a user to opt in to to say "Use the init.vim value only".

**Comment by badosu** [comment]
I agree, I use the same colourscheme on terminal vim and I have to set it at Oni for that to be enforced. It would be good if we could make Oni inherit you vim's preferences and avoiding overriding too much so that users that like to have a consistent environment can benefit.

**Comment by badosu** [comment]
@CrossR Added your suggestion to the minimal wiki page, using it now, thanks!
